components/form-control/README.md

```diff
@@ -0,0 +1,341 @@
+~~~~
+
+* import the gameface-form-control component:
+
+~~~~{.html}
+<script src="./node_modules/gameface-form-control/umd/gameface-form-control.production.min.js"></script>
+~~~~
+
+This is all! Load the file in Gameface to see the gameface-form-control.
+
+If you wish to import the modules using JavaScript you can remove the script tags
+which import the components and the gameface-form-control from the node_modules folder and import them like this:
+
+~~~~{.js}
+import components from 'coherent-gameface-components';
+import gameface-form-control from 'gameface-form-control';
+~~~~
+
+Note that this approach requires a module bundler like [Webpack](https://webpack.js.org/) or [Rollup](https://rollupjs.org/guide/en/) to resolve the
+modules from the node_modules folder. Alternatively, you can import them directly from node_modules:
+
+~~~~{.js}
+import components from './node_modules/coherent-gameface-components/umd/components.production.min.js';
+import gameface-form-control from './node_modules/gameface-form-control/umd/gameface-form-control.production.min.js';
+~~~~
+
+## Usage with CJS modules:
+
+* Import the components library:
+
+~~~~{.js}
+const components = require('coherent-gameface-components');
+const gameface-form-control = require('gameface-form-control');
+~~~~
+
+The CommonJS(CJS) modules are used in a NodeJS environment, be sure to use a module
+bundler to use them in a browser.
+
+# Form control attributes
+
+| Attribute | Required | Values        | Default value | Usage                                                                                                                                                                            |
+| --------- | -------- | ------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `method`  | Yes      | `GET`, `POST` | N/A           | Used to specify the HTTP method that will be used to send data when the form is submitted. **If the attribute is not added or with no valid value the form won't be published.** |
+| `action`  | No       | Any valid URL | `./`          | Used to specify where the form will send the data. **If the attribute is not added the data will be sent to the current page URL.**                                              |
+
+### Examples
+
+* `<gameface-form-control method="GET" action="http://localhost:12345/login"></gameface-form-control>` - Submitting this form will create a `GET` request with the form data and send it to the server that is available on `http://localhost:12345/login`.
+* `<gameface-form-control method="POST"></gameface-form-control>` - Submitting this form will create a `POST` request with the form data and send it to the current page location.
+* `<gameface-form-control></gameface-form-control>` - Submitting this form will produce a warning and won't the request will be aborted.
```

Comment:
Submitting this form will produce a warning and won't the request will be aborted. - "...and the request will be aborted" maybe?
https://www.reddit.com/r/memes/comments/qtndvq/1_hour_before_due_date_be_like/

Reply:
Ahh Read Kalo, read ... :D Fixed